cfme/tests/ansible/test_embedded_ansible_crud.py

```diff
@@ -34,44 +34,19 @@ def test_embedded_ansible_enable(embedded_appliance):
     assert wait_for(lambda: embedded_appliance.is_embedded_ansible_running, num_sec=30)
     assert wait_for(lambda: embedded_appliance.rabbitmq_server.running, num_sec=30)
     assert wait_for(lambda: embedded_appliance.nginx.running, num_sec=30)
+    assert wait_for(
+        lambda: embedded_appliance.supervisord.status == "active", num_sec=180
+    )
+    assert wait_for(
+        lambda: embedded_appliance.rabbitmq_server.status == "active", num_sec=80
```

Comment:
Q: after `status` check do you think above `rabbitmq_server.running` check required?

Reply:
Earlier `.running` used to return `True` or `False`, so my intention to check existing status of the service.